components/ConvertSteps/useStepLayout.js

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect, useRef } from 'react'
```

Comment:
Not sure, but could [`use-viewport`](https://github.com/aragon/use-viewport) be a good substitute?

Reply:
Interesting. I had actually forgotten about `use-viewport`. While it will work in this context I think in the future we are going to need a way to compare an outer boundary with an inner where the viewport isn't the right measurement to use e.g. Stepper inside a modal which is smaller than the viewport.

I see it's been raised as an improvement https://github.com/aragon/use-viewport/issues/1 would be interested to explore how we can use observers to achieve it in the future.

I'll swap this out now as it should be a slot in replacement 👍